docs/DECISIONS.md

```diff
@@ -104,7 +104,7 @@ Having all the services defined in a single file would make it difficult to navi
 
 ### docker-aliases
 
-While the config to run each service with docker-compose is service-specific, there are common elements between each stack. In order to reduce this duplication, **govuk-docker uses a x-service extension for each service**.
+While the config to run each service with docker-compose is service-specific, there are common elements between each stack. In order to reduce this duplication, **govuk-docker uses x-my_service section to define the common elements and relies on YAML aliases to embed the common elements in each stack**.
```

Comment:
Just a minor tweak: 'section' -> 'sections'. Also, I'd suggest the new sentence ending is not in bold, as it's not really part of the decision, just more detail about the mechanism.